### ISSUE TITLE
Color contrast is pretty low across the board

### ISSUE THREAD

**Issue by juliusv** [vector-im/vector-web#2300]
I could include a screenshot here, but I guess you all know what Vector / Riot looks like :) Comparing it to Slack, the color contrast is pretty low and is straining my eyes a bit. That goes mainly for the UI bits, but even the main chat message text is `#454545`, whereas Slack's is `#2c2d30`. The left sidebar is worse though, with its grey text on light green background. Any chance we could get more contrast as a default?

**Comment by nshopik** [comment]
I've noticed that text kinda thinner, which make harder to keep focus on main chat as other elements seems same size and low contrast. 

Left panel is suppose to be visible separation but it feels like its part of chat. Same goes for usernames making them grey text make it worse, should be same color but bold, like in slack and many irc apps, especially when we don't have avatar generation(random image if nothing set) you can easily lost track in busy room.

Also not sure why time is hidden by default for all messages but last one, making user unnecessary move mouse just to see time of message.

These small things make it overall low contrast and harder to keep focus on main content as your eyes keep cling for some UI elements instead chat. I know some stuff is personal preference but I've tried to be constructive about it.

**Comment by juliusv** [comment]
As an example, I changed the main chat colors, font weight of names, and whitespace between messages a bit locally via the Chrome inspector and it is already sooo much more readable:

![contrast](https://cloud.githubusercontent.com/assets/538008/18747525/26b11ed2-80ce-11e6-86ca-1d851d5cad74.png)

**Comment by richvdh** [comment]
Related to #1635

**Comment by taw00** [comment]
Related to <https://github.com/vector-im/riot-web/issues/2460> and <https://github.com/vector-im/riot-web/issues/2865>

**Comment by subiol** [comment]
+1 the font is very light and makes the app hard to use

**Comment by ishitatsuyuki** [comment]
I would happy to see more darkened default colors for the meanwhile.

**Comment by ara4n** [comment]
PRs for contrasty themes would be enormously appreciated. https://github.com/vector-im/riot-web/blob/master/docs/theming.md is the doc of how to add themes.

**Comment by yiskah** [comment]
I hunted down this github just to cosign that the contrast ratios are way too low and it's super inaccessible. was unable to keep using for more than an hour due to eye strain. especially in dark mode i couldn't even see the unlocked icon. https://imgur.com/a/y9mmp

use this contrast checking tool: http://webaim.org/resources/contrastchecker/

you want a 4.5:1 ratio

**Comment by hoodiek** [comment]
plus one to yiskah, i also cant really tell when a room has new messages due to the contrast issue

**Comment by hoodiek** [comment]
i'm not a huge css coder but i'll see if i can do anything about it myself, later tonight or tomorrow

**Comment by tessgadwa** [comment]
Are there any naming conventions or guidelines for theme names?

**Comment by ara4n** [comment]
> Are there any naming conventions or guidelines for theme names?

no; i'd suggest light-contrasty and dark-contrasty :)

**Comment by tessgadwa** [comment]
Uploaded a pull request, but probably best to ignore for now.  Allan Day
and I are working on some further refinements to the design.  Visit
#designUX on Matrix for screenshots.

On Mar 10, 2017 4:57 PM, "Matthew Hodgson" <notifications@github.com> wrote:

> Are there any naming conventions or guidelines for theme names?

no; i'd suggest light-contrasty and dark-contrasty :)

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/vector-im/riot-web/issues/2300#issuecomment-285825127>,
or mute the thread

**Comment by tessgadwa** [comment]
Just resubmitted PR for light high contrast theme, incorporating feedback from graphic designer Alan Day. On to dark high contrast theme next.
